lib/util/fs.cpp

```diff
@@ -0,0 +1,17 @@
+#include "cppship/util/fs.h"
+#include <cstdlib>
+
+void cppship::create_if_not_exist(const fs::path& path)
+{
+    if (fs::exists(path)) {
+        return;
+    }
+
+    if (path.has_parent_path()) {
+        create_if_not_exist(path.parent_path());
+    }
+
+    fs::create_directory(path);
```

Comment:
use `create_directories`?

Reply:
Yep, not familiar with the filesystem library.